### ISSUE TITLE
R.I.P 

### ISSUE THREAD

**Issue by yanyixing19** [leixiaohua1020/simplest_ffmpeg_mobile#54]

**Comment by haseebpvt** [comment]
What happen?!

**Comment by dbof10** [comment]
Did he die?

**Comment by haseebpvt** [comment]
@dbof10 Yes. Here is the story: http://en.people.cn/n3/2016/0805/c90000-9095938.html

**Comment by haseebpvt** [comment]
Lei Xiaohua, 25, a PhD candidate at Communication University of China (CUC), died suddenly at midnight on July 17. The young man's body was found by sanitation workers the next morning.

Lei's family believes that he died from overwork, as he regularly buried himself in his projects at the lab. However, the university countered that since Lei's family did not request an autopsy, the cause of death cannot be confirmed.

According to a video provided by the university, Lei went into a second-floor lab at 9:30 p.m. on the night of July 16. From 11 p.m. onwards he began to go to the bathroom frequently. Then he walked to fifth floor, where he was seen vomiting.

Lei's mother said her son never had health problems before. Lei had even participated in a half-marathon race. However, one week before his death, Lei had been working in the lab most nights from 9 p.m. to midnight.

An investigation by the Chaoyang Public Security Bureau determined that Lei's death may have been sudden, but it does not constitute a criminal case.

Lei's mother said the family had been in touch with the university in the wake of Lei's death, but that the university had not offered any clarity or comfort. Instead, the university claimed that Lei was drunk. A CUC administrator also explained that the family needed to ask relevant departments for information. Only certain departments are authorized to give an official reply.

From communications via telephone and WeChat, several of Lei's peers said that he seemed to be under heavy pressure from his work and studies.

Source: http://en.people.cn/n3/2016/0805/c90000-9095938.html

**R.I.P**
